### DISCUSSION TITLE
Escape Routes on Vercel

### DISCUSSION THREAD

**Discussion by ramyjaber1** [vercel/community#466] [Help] [answered]
hello , sorry i don't know how to explain that 
i have a website justriskit.com built using nextjs deployed on vercel i also have word press on the domain which is justriskit.com/wp-admin but whenever i tried to access the link i got 404 page and i know that vercel does not support htaccess file and i should use vercel.json but what should i write into it ?

**Comment by Pieparker** [comment, marked as answer]
We have a [guide on custom 404 page handling](https://vercel.com/guides/custom-404-page) which will explain things better than I possibly could 😄. Check this out and let us know if you have any follow-up questions!
